fio_jobfiles/run_fio.sh

```diff
@@ -1,12 +1,46 @@
 #!/bin/sh
-export DATA_DIR=/data/fio_datafiles/k8s
-export WORK_DIR=/data/$JOB_NAME/$POD_NAME
-mkdir -p $WORK_DIR
-let i=128
-let lim=16*1024*1024
-while [ $i -le $lim ]
-do
-    echo $i
-    fio /fio_jobfiles/read_random.fio --runtime=30 --directory=$DATA_DIR --output-format=json+ --blocksize=$i > $WORK_DIR/$i.json
-    let i=2*i
+set -ux
+
+prepare () {
+  if [ "$FIO_JOB" = "write" ]; then
+    export SCRATCH_DIR=/data/fio_write/$POD_NAME
```

Comment:
Hard-coded path here - would prefer it to be parameterised, particularly if this script was used outside of a Kubernetes environment.